sdk/python/feast/feature_server.py

```diff
@@ -69,6 +68,13 @@ class MaterializeIncrementalRequest(BaseModel):
     feature_views: Optional[List[str]] = None
 
 
+class GetOnlineFeaturesRequest(BaseModel):
+    entities: Dict[str, List[Any]]
+    feature_service: Optional[str] = None
+    features: Optional[List[str]] = None
```

Comment:
could (should) add validation on the model that throws if both are set. didnt do it here, bc it would be technically breaking. the code works just picks the `feature_service` value and ignore `features`. this is an unintuitive (successful) result, seems like it would make more sense to return `422` and say, "gotta pick one". maybe flag it for 1.0? @franciscojavierarceo